egs/wsj/s5/tensor_2048_relu.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2016    Vincent Renkens
```

Comment:
Please don't put files in the top-level experimental directory.
They should be in local/tensorflow/ or something like that.